File: examples/performance/objects/Objects.java
class Objects {
    static class Box {
        public final int number;

        public Box(int number) {
            this.number = number;
        }
    }

	public static void main(String[] args) {
	    for(int i = 0; i < 100_000_000; i++) {
	        var o = new Box(i);
	    }
	}
}

File: examples/performance/objects/objects.cpp
// g++ objects.cpp -o objects++ -Ofast

class Box
{
public:
    const int number = 0;
    Box(int number) : number(number) {}
};

int main()
{
    for (int i = 0; i < 100000000; i++)
    {
        auto o = Box(i);
    }

    return 0;
}
